uobtheatre/payments/models.py

```diff
@@ -281,38 +293,37 @@ def refund(self, refund_provider: RefundProvider = None):
         refund_provider.refund(self)
 
 
-TOTAL_PROVIDER_FEE = Coalesce(
-    Sum(
-        "provider_fee",
-    ),
-    0,
-)
-NET_TOTAL = Sum("value")
-NET_CARD_TOTAL = Sum("value", filter=(~Q(provider_name=Cash.name)))
-TOTAL_SALES = Sum("value", filter=Q(type=Transaction.Type.PAYMENT))
-TOTAL_CARD_SALES = Sum(
-    "value", filter=(~Q(provider_name=Cash.name) & Q(type=Transaction.Type.PAYMENT))
-)
-TOTAL_REFUNDS = Sum("value", filter=Q(type=Transaction.Type.REFUND))
-TOTAL_CARD_REFUNDS = Sum(
-    "value", filter=(~Q(provider_name=Cash.name) & Q(type=Transaction.Type.REFUND))
-)
-APP_FEE = Coalesce(Sum("app_fee"), 0)
-
-SALE_BREAKDOWN_ANNOTATIONS: dict[str, Any] = {
-    # Gross Income
-    "net_income": NET_TOTAL,
-    "net_card_income": NET_CARD_TOTAL,
-    # Total Purchases / Sales
-    "total_sales": TOTAL_SALES,
-    "total_card_sales": TOTAL_CARD_SALES,
-    # Total Refunds
-    "total_refunds": TOTAL_REFUNDS,
-    "total_card_refunds": TOTAL_CARD_REFUNDS,
-    # Gross Amount charged by the payment provider (square) for these payments
-    "provider_payment_value": TOTAL_PROVIDER_FEE,
-    # Amount we take from net payment - provider cut
-    "app_payment_value": APP_FEE - TOTAL_PROVIDER_FEE,
-    "society_transfer_value": NET_CARD_TOTAL - APP_FEE,
-    "society_revenue": NET_TOTAL - APP_FEE,
-}
```

Comment:
as we are re-doing this please can we not lose docs for this - james and I struggled enough when sorting out the money this year that when it comes round again there should be detailed and descriptive docs for each one of these (even if it is self explanatory)

Reply:
unless the renaming of these terms is in a different pr then ill shush

Reply:
Ive updated the comment on the methods in payable. Can you check these @streth11